README.md

```diff
@@ -3,64 +3,77 @@ android-player-samples
 
```

Comment:
We should revert most of the changes to this file, namely the references to the ExoPlayer2 branch back to master, since that's where the Exo2 samples will live now.